src/uu/tail/src/tail.rs

```diff
@@ -169,14 +170,15 @@ fn tail_file(
         }
         observer.add_bad_path(path, input.display_name.as_str(), false)?;
     } else if input.is_tailable() {
-        let metadata = path.metadata().ok();
         match File::open(path) {
             Ok(mut file) => {
+                let st = file.metadata()?;
```

Comment:
What does `st` stand for?

Reply:
AFAIK, st is used as the underlying syscall is stat()

Reply:
Ah, ok, thanks for the explanation!